superset-frontend/spec/javascripts/sqllab/ShareSqlLabQuery_spec.jsx

```diff
@@ -27,6 +27,7 @@ import { render, screen, act } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import userEvent from '@testing-library/user-event';
 import * as utils from 'src/utils/common';
+import { hexToRgb } from 'src/utils/colorUtils';
```

Comment:
Intriguing... I see this util is being used for testing only thus far. I wonder if it should be in a testUtils folder if that's the intention for it.

I could also see it being useful elsewhere though in the future... and maybe even extended handle hex+alpha -> RGBa. A similar util for HSL(a) could also come in handy from time to time.

Reply:
> I could also see it being useful elsewhere though in the future... and maybe even extended handle hex+alpha -> RGBa. A similar util for HSL(a) could also come in handy from time to time.

That was the intention. A global utility (not only for tests), starting with RGB to HEX and HEX to RGB conversions. I hope to see it with more functions in the future.